crates/node-builder/src/builder.rs

```diff
@@ -508,6 +531,8 @@ where
         let NodeComponents { transaction_pool, network, payload_builder } =
             components_builder.build_components(&ctx).await?;
 
+        // TODO(alexey): launch ExExs and consume their events
+
```

Comment:
maybe this should happen after rpc,
unsure if there are any race conditions though, don't think so

Reply:
don't think so